.github/workflows/stats_lambda.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow executes a Lamdba function that recording metrics for each successfully merged PR.
+#
+# You can adjust the behavior by modifying this file.
```

Comment:
I would remove the whole "You can adjust..." line

Reply:
Done